test/TestApps/PushNotificationsTestApp/main.cpp

```diff
@@ -312,15 +364,24 @@ int main() try
         ::Test::Bootstrap::CleanupBootstrap();
     });
 
-
     ::Test::Bootstrap::SetupBootstrap();
 
-    PushNotificationActivationInfo info(
-        PushNotificationRegistrationOptions::PushTrigger | PushNotificationRegistrationOptions::ComActivator,
-        winrt::guid(c_comServerId)); // same clsid as app manifest
+    // TODO: Register activators only when the unit test requires it
+    if (PushNotificationManager::IsActivatorSupported(PushNotificationRegistrationOptions::ComActivator))
+    {
+        PushNotificationActivationInfo info(
+            PushNotificationRegistrationOptions::PushTrigger | PushNotificationRegistrationOptions::ComActivator,
+            winrt::guid(c_comServerId)); // same clsid as app manifest
 
-    g_appToken = PushNotificationManager::RegisterActivator(info);
-    
+        g_appToken = PushNotificationManager::RegisterActivator(info);
+    }
+    /* TODO: Register ProtocolActivator for unpackaged applications
```

Comment:
Nit: TODO: Register ProtocolActivator for unpackaged applications or Packaged Applications when COM activation is unsupported